.github/workflows/pkgci_test_amd_mi300.yml

```diff
@@ -17,6 +17,7 @@ on:
         type: string
         default: ""
 
+# test
 jobs:
   test:
     runs-on: nodai-amdgpu-mi300-x86-64
```

Comment:
Huh... this is taking a while to start the job on a runner:

> Started 8m 7s ago
> Job is about to start running on the runner: iree-mi300-3 (repository)

![image](https://github.com/user-attachments/assets/e7ac917a-a077-4044-a33d-ee6c997a500c)

https://github.com/iree-org/iree/actions/runs/10498923807/job/29085054817?pr=18318

🤔  weird... the machine seems to be up, `iree-mi300` and `iree-mi300-8` are both running jobs

Reply:
Yeah, it's one of those stalls :( Trying to cancel and rerun